src/main/java/stepsdesign/PipelineSteps.java

```diff
@@ -659,4 +665,18 @@ public void clickOnCreateButton () {
   public void verifyInvalidConnectionErrorMessage(String errorMessageLocation) {
     CdfConnectionActions.verifyConnectionErrorMessage(errorMessageLocation);
   }
+
+  @Then("Click on the Plus Green Button to import the pipelines")
+  public void clickOnPlusGreenButton () {
+    CdfConnectionActions.clickOnAddEntityButton();
+  }
+
+  @Then("Select the json files for importing the pipelines for the plugin {string}")
+  public void selectJSONFiles(String path) throws URISyntaxException {
+    CdfConnectionActions.importJsonFiles(PluginPropertyUtils.pluginProp(path));
+  }
+
+  @When("Click on the fix all button if displayed")
```

Comment:
You are using the fixAllButton method when you are uploading the pipeline I don't think you need a step for this.

Reply:
resolved